django_app/redbox_app/redbox_core/consumers.py

```diff
@@ -76,19 +77,16 @@ async def receive(self, text_data=None, bytes_data=None):
         user_message_text: str = data.get("message", "")
         selected_file_uuids: Sequence[UUID] = [UUID(u) for u in data.get("selectedFiles", [])]
         user: User = self.scope.get("user")
-        chat_backend = data.get("llm")
         temperature = data.get("temperature")
 
         if session_id := data.get("sessionId"):
             session = await Chat.objects.aget(id=session_id)
-            logger.info("updating: chat_backend=%s -> ai_settings=%s", session.chat_backend, chat_backend)
-            session.chat_backend = chat_backend
+            if llm := data.get("llm"):
+                session.chat_backend = await ChatLLMBackend.objects.aget(name=llm)
             session.temperature = temperature
             await session.asave()
         else:
-            session = await Chat.objects.acreate(
-                name=user_message_text[: settings.CHAT_TITLE_LENGTH], user=user, chat_backend=chat_backend
-            )
+            session = await Chat.objects.acreate(name=user_message_text[: settings.CHAT_TITLE_LENGTH], user=user)
```

Comment:
Ditto above should we log that we are creating a new chat as debug?

Reply:
[79de87e](https://github.com/i-dot-ai/redbox/pull/1084/commits/79de87e32060e2b01b87980d6e42e86388f7effa)